site/includes/dependencies.pug

```diff
@@ -18,10 +18,10 @@ link(rel='stylesheet', type='text/css', href='../components/vars/spectrum-large.
 each dep in dependencyOrder.filter((dep) => dep != 'vars' )
   link(rel='stylesheet', type='text/css', href='../components/' + dep + '/index-vars.css', data-dependency=dep )
 
-link(rel='stylesheet', type='text/css', href='css/site.css')
 link(rel='stylesheet', type='text/css', href='css/docs.css')
```

Comment:
So do we still need `docs.css` @jianliao ?

Reply:
@lazd Looks like we need both css/site.css and css/docs.css. Without css/site.css, all the typography-deprecated failed ( pls check above VRT result). Without css/docs.css, Action Menu failed ( I checked it locally ).

Reply:
Now you can go directly to the fail scenario by clicking the test.
![image](https://user-images.githubusercontent.com/1207520/88111176-4f8b8a80-cb62-11ea-8c97-1e61934926be.png)

Reply:
Interesting, ok. Kinda weird that those caused failures though.